SITCOMTN-076.tex

```diff
@@ -49,6 +53,325 @@
+%as determined by the Project and reviewed by [the SAC?, Ops Data Release Board?]
+%These would be labeled as Preliminary Data, with
+%There will be caveats and preliminary information about calibration.
+% and no Project support for use until DP2.
+%After the Data Preview 2 release
+%Within 6 months following the start of LSST, PVIs will be released within the standard time window for steady-state LSST operations (see \citedsp{RTN-011}).
+Within 6 months following the start of LSST, PVIs and source catalogs will be released within the standard time window for steady-state LSST operations (see \citedsp{RTN-011}).
+
+\section{Commissioning Era Information Sharing Policies}
+\label{policy}
+
+\subsection{Summary}
+
+%Project team members must follow policies for embargoed data products described in the Vera C. Rubin Observatory Data Policy \citedsp{RDO-013} and Project Publication Policy \citedsp{LPM-162}.
+%, and Rubin Observatory Data Security Standards Implementation (DMTN-199).
+All pixel images and representations of pixel images of any size field of view, including individual visit images, coadd images, and difference images based on LSSTCam commissioning on-sky observations are embargoed for a period of at least 30 days, with the exception of alert postage stamp images.
+
+During the period from installation of LSSTCam on the telescope to the release of Data Preview 2, all communications (including informal discussion) regarding proprietary engineering and on-sky data with LSSTCam are internal to Rubin Project by default.
+Outward-facing communications, including all on-sky LSSTCam images, are reviewed by SIT-Com leadership, the Rubin Celebrations Organizing Committee (RCOC) and its relevant subgroups, and the Communications team, and are approved for release by the Project Director or designated alternate.
+
+Example outward-facing communications:
+
+\begin{itemize}
+
+  \item Weekly digests and News stories on \href{https://rubinobservatory.org/}{Rubin Observatory webpages}
+
+  \item \href{https://www.lsst.org/about/project-status}{Published project status webpage}
+
+  \item Rubin Observatory press releases / media events
+
+  \item Released photographs, plots, and on-sky images
+
+  \item Approved technotes (see Section \ref{technotes})
+
+\end{itemize}
+
+Free and unfettered communication among Project team members is essential for commissioning success.
+%For proprietary data,
+Project team members are required to use only approved Project tools, platforms, and processes for communication, data access and analysis, documentation, software development, work management, etc.
+Derived data products are not subject to those restrictions, but are embargoed until approval for release (see Section \ref{technotes}).
+In practice, we expect most work done by the Project team on the commissioning data to be done within protected directories at the Rubin US Data Facility at SLAC.
```

Comment:
Is it worth being explicit about how we'll use slack?  It's an essential part of project culture, but we'll have to be careful not to let data leak.

Reply:
Added a sentence to give more specificity to "Approved Project tools", including a mention of Slack. To my knowledge, we have not yet made a decision on details of Slack usage.